src/exploreBDD.test.js

```diff
@@ -0,0 +1,203 @@
+import { expect } from 'chai';
+
+describe('Basic Assertions', () => {
+    it('Number_equal', () => {
+        expect(2 + 3).to.equal(5)
+    });
+
+    it('Number_not_equal', () => {
+        expect(2 + 3).to.not.equal(6)
+    });
+
+    it('String_equal', () => {
+        expect('DongThin').to.equal('DongThin')
+    });
+
+    it('Array_Deep equal', () => {
+        expect(['DT', 'Al']).to.deep.equal(['DT', 'Al']);
+    });
+
+    it('Obj_Not deep equal', () => {
+        const actual = { name: 'Kien', age: 29 };
+        const expected = { name: 'Thin', age: 25 };
+        expect(actual).to.not.deep.equal(expected);
+    });
+
+    function sayHello(name) {
+        return `Hello, ${name}`;
+    }
+
+    it('Obj_Equal', () => {
+        function getUser() {
+            return {
+                name: 'DongThin',
+                age: 18,
+            }
+        }
+        const user = getUser();
+        expect(user).to.have.property('name').that.is.a('string');
+        expect(user).to.have.property('age').that.is.a('number');
+        expect(user.age).to.be.within(18, 30);
+        expect(user).to.deep.equal({ name: 'DongThin', age: 18, })
+    })
+
+    it('Literal Expression_Equal', () => {
+        const name = 'DongThin';
+        expect(sayHello(name)).to.equal('Hello, DongThin');
+    });
+
+})
+describe('Booleans/Truthiness', () => {
+    it('Booolean values_True', () => {
+        expect(true).to.equal(true);
+    })
+    it('Booolean values_False', () => {
+        expect(false).to.equal(false);
+    })
+    it('Truthy_True', () => {
+        expect(1).to.be.ok;
+    })
+
+    it('Falsy_False', () => {
+        expect(0).to.be.not.ok;
+        expect('').to.be.not.ok;
+    })
```

Comment:
Make sure you take note these IMPORTANT Falsy cheat sheet: https://developer.mozilla.org/en-US/docs/Glossary/Falsy.